pdns/recursordist/html/local-2022.js

```diff
@@ -1,13 +1,44 @@
 "use strict";
 
-// var moment= require('moment');
+const fetchConfig = {
+    baseURL: window.location,
+    mode: 'same-origin',
+    headers: {'Accept': 'application/json'},
+};
+/*
+// Useful for development of the embedded webserver files.
+const fetchConfig = {
+    baseURL: 'http://127.0.0.1:8083/',
+    mode: 'cors',
+    headers: {'Accept': 'application/json', 'X-API-Key': 'changeme'},
+};
+*/
+
 var gdata = {};
 
-$(document).ready(function () {
-    $.ajaxSetup({cache: false});
+function get_json(url, params) {
+    const realURL = new URL(url + '?' + (new URLSearchParams(params)).toString(), fetchConfig.baseURL);
```

Comment:
This works in our usecases but could easily break, could you rather use `realURL.append` on each key/value pairs in `params` please?

Something such as:
```js
const realURL = new URL(url, fetchConfig.baseURL);
for (const [k, v] of Object.entries(params)) {
    realURL.append(k, v);
}
```

Reply:
done!